Improve quickstart page to more involved example

Globegitter opened the pull request:
Since it would have been useful for me to have a more complex example I thought I would create one that is quite similar to one of our current use cases. It still doesn't show `ConnectionFields`, but maybe that would be a bit too much anyway for people who are completely new to graphql and/or relay.

syrusakbary commented:
Thats a good advice! However a quickstart should need to have everything needed for "warming up" and "working" doing copy&paste.
If you could make the example work with just one file (not importing `somewhere`) and working I will love to merge it.
PS: Maybe doing a very simple `dict` memory storage solution will work :wink:

Globegitter commented:
`user_id` is a field in the `User` Object. The reason we use separate `id` and `..._id` in our apps is so we can display our actual IDs. But happy to completely remove it and write as you suggested.

Otherwise addressed your comments :)

syrusakbary commented on quickstart.md:
```diff
@@ -33,20 +33,101 @@ pip install graphene
+
+USER_DATA = [
+    {
+        user_id = '1'
```
This syntax is not python valid :( (have to be key: value)

Globegitter commented:
Sorry about that - my head was clearly on kwargs.

ptz0n commented on quickstart.md:
```diff
@@ -33,20 +33,101 @@ pip install graphene
+                            fullName=graphene.String(),
+                            sortKey=graphene.String(),
+                            sortDirection=graphene.String())
+        user = relay.NodeField(Users)
```
I cannot find where you have defined the `Users` class.

jhgg replied:
Typo. Should be `User` I think.

syrusakbary commented:
I have to see if everything works, and if is the case I will approve soon! @Globegitter

ptz0n commented:
Please have a look in Python 2.7 as well. I see alot of syntax errors in the code right now.

ptz0n commented:
@Globegitter is this really working for you? Output from [`test.py`](https://gist.github.com/ptz0n/465eca6fa2ab1161fbbd) is `None`.

Works if you change `sort_directon` to `sort_direction`. :+1:

ptz0n commented on quickstart.md:
```diff
@@ -33,20 +33,101 @@ pip install graphene
+    sort_key = args.get('sortKey', None)
+    sort_direction = args.get('sortDirection', None)
+    reversed = False
+    if sort_directon and sort_directon == 'DESC':
```
This one, should be `sort_direction`.

Globegitter commented:
@ptz0n Yeah was working for me, but rather than copy&pasting the fixes I just typed them. And well seemed to have been a bit blind. Anyway should be good now.

ptz0n commented:
Would be nice if you run `flake8` on the new example code.

Globegitter commented:
Did just that @ptz0n :)

adamcharnock commented:
Just thought I'd drop a note here to say that I plan to do a lot of work on the Django quickstart guide as part of PR #60.

syrusakbary commented:
Closing this issue as the Quickstart guide docs are now more detailed now.
Please reopen if you feel that something have to be changed.
